.changelog/4694.yml

```diff
@@ -0,0 +1,4 @@
+changes:
+- description: Fixed an issue where setup-env would fail when FILE_PATHS argument was missing.
+  type: internal
```

Comment:
Internal?

Reply:
Yes, since it wasn't release and it was caught internally, I don;t think it should go out as a fix.